app/assets/javascripts/modules/track-responses.js

```diff
@@ -10,14 +10,25 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.$module.addEventListener('submit', this.handleFormSubmit.bind(this))
   }
 
+  TrackResponses.prototype.filterLabels = function () {
+    return ['vaccination-status']
```

Comment:
This returns an array of only one value. Are we expecting to have to expand this at some point, or could it return a single string? If we're expecting to expand it, can we have a test that checks the code works if an array of more than one value is returned from this function?

Reply:
Ah! You caught me - I was thinking this wouldn't be the only one over time - but currently it is.  I'll update to a single string for now.

Reply:
Updated for single value